.taskcluster.yml

```diff
@@ -0,0 +1,33 @@
+version: 0
+metadata:
+  name: "TaskCluster GitHub Tests"
```

Comment:
We should change the name and description to be about taskcluster-lib-validate instead of taskcluster-github